### ISSUE TITLE
what about directly ablate the image or just use a BLACK image?

### ISSUE THREAD

**Issue by DingchenYang99** [DAMO-NLP-SG/VCD#2]
greetings,
I think VCD is very interesting. I kept wondering whether using a pure black image (zeros tensor) or directly ablating the noised image could help as well, i.e. make the model blind to visual inputs.  🤔 As Figure.2 in the main paper states that "The ground-truth color ”black” diminishes in probability (logp(y|x,v0)) as the distortion escalates, making LVLMs over-reliant on the language priors from LLM pre-training that typically associate bananas with being yellow or green".

**Comment by LengSicong** [comment]
Hi, thanks for your interest in our work. 
We have explored contrasting the model's output distribution with pure Gaussian noise or all zero tensors. However, we found that this aggressive strategy may mislead the model outputs in some scenarios where visual information is crucial. For example, when prompting LVLMs to caption a given image, the output with a completely distorted image may provide unreasonable answers, and contrasting with this output distribution would not benefit LVLMs' content generation.
